src/Components/NavBar/NavBarLoggedOutActions.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, Flex, Spacer } from "@artsy/palette"
+import { ModalType } from "Components/Authentication/Types"
+import { ContextModule, Intent } from "@artsy/cohesion"
+import { useTranslation } from "react-i18next"
+import { useAuthDialogAdapter } from "Components/AuthDialog"
+
+export const NavBarLoggedOutActions = () => {
+  const { t } = useTranslation()
+
+  const { showAuthDialog } = useAuthDialogAdapter()
+
+  return (
+    <Flex alignItems="center">
+      <Button
+        variant="secondaryBlack"
+        size="small"
+        onClick={() => {
+          showAuthDialog({
+            current: {
+              mode: "Login",
+              analytics: {
+                contextModule: ContextModule.header,
+                intent: Intent.login,
+              },
+            },
+            legacy: {
+              mode: ModalType.login,
+              intent: Intent.login,
+              contextModule: ContextModule.header,
+              copy: "Log in to collect art by the world’s leading artists",
+            },
+          })
+        }}
```

Comment:
cc @damassi — I'm a little torn about this. I want to break the existing API for two reasons:

* The flat arguments are confusing to me. Between what's used for analytics and what is a configuration option for the actual dialog. Like right now `intent` is used specifically for analytics vs `afterSignUpAction/afterAuthAction`, which actually runs an [`AuthIntent`](https://github.com/artsy/force/blob/main/src/Utils/Hooks/useAuthIntent/index.tsx)
* The way title works is slightly different, accepting an optional function which can be used to configure the string based on view mode. (The existing production implementation actually breaks, where it shows the initial copy string, you change views and it switches it to a generic, then if you swap back to the other view it uses the default for that view.)

Reply:
Updated API looks much better to me assuming there isn't fallout elsewhere 👍